.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "e2e_accept/appLib/laTrappe"]
+	path = e2e_accept/appLib/laTrappe
+	url = git@github.com:kulicuu/laTrappe.git
```

Comment:
so people don't need to have it downloaded with the clone.  Originally I was going to add the app (laTrappe) into the repository properly, but this is _much much_ better.